### PR TITLE
build fixes

### PR DESCRIPTION
**Pull request by dheerajkd30**

### CODE REVIEW

File: x/asset/keeper/asset.go [hunk 1]
```diff
@@ -137,3 +137,12 @@ func (k *Keeper) DeleteAssetForDenom(ctx sdk.Context, denom string) {
 
 	store.Delete(key)
 }
+
+func (k *Keeper) GetPriceForAsset(ctx sdk.Context, id uint64) (uint64, bool) {
```
Comment (prashantkrv2020) [inline x/asset/keeper/asset.go]: push this function to oracle module as well

File: x/asset/keeper/keeper.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ type Keeper struct {
 	channel expected.ChannelKeeper
 	port    expected.PortKeeper
 	scoped  expected.ScopedKeeper
+	oracle expected.OracleKeeper
```
Comment (prashantkrv2020) [inline x/asset/keeper/keeper.go]: asset module wont require this. only vault module will  require both asset and oracle

File: x/cdp/keeper/keeper.go [hunk 3]
```diff
@@ -12,17 +12,19 @@ import (
...
 }
 
-func NewKeeper(cdc codec.BinaryCodec, key sdk.StoreKey, asset expected.AssetKeeper) Keeper {
+func NewKeeper(cdc codec.BinaryCodec, key sdk.StoreKey, asset expected.AssetKeeper, oracle expected.OracleKeeper) Keeper {
 	return Keeper{
-		cdc:   cdc,
-		key:   key,
-		asset: asset,
+		cdc:    cdc,
+		key:    key,
```
Comment (prashantkrv2020) [inline x/cdp/keeper/keeper.go:25]: bank keeper not present

Comment (dheerajkd30) [reply x/cdp/keeper/keeper.go:25]: fixing it in another pr, with vault changes.

**Pull request closed without merging**